src/data/make_dataset.py

```diff
@@ -1,3 +1,79 @@
+import os
+import zipfile
+from typing import Union
+import pandas as pd
+from kaggle.api.kaggle_api_extended import KaggleApi
+from sklearn.model_selection import train_test_split
+
+def download_dataset(dataset_name: str, destination_path: str) -> None:
```

Comment:
If we want to store data in google drive maybe it is not necessary to download it through kaggle? Once we all have access to google drive, each of us can easily download the data from there.

So my suggestion would be to download the raw data, add it to the folder, configure dvc, push changes, send link to others. Does it make sense? What do you think?

Reply:
Makes sense, I'll configure dvc and remove the kaggle code.